src/class-elements/rs-private-getter-alt.case

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2018 Kubilay Kahveci (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Valid PrivateName as private getter
+info: |
+
+  ClassElement :
+    MethodDefinition
+    ...
+    ;
+
+  MethodDefinition :
+    ...
+    get ClassElementName ( ){ FunctionBody }
+    ...
+
+  ClassElementName :
+    PropertyName
+    PrivateName
+
+  PrivateName ::
+    # IdentifierName
+
+  IdentifierName ::
+    IdentifierStart
+    IdentifierName IdentifierPart
+
+  IdentifierStart ::
+    UnicodeIDStart
+    $
+    _
+    \ UnicodeEscapeSequence
+
+  IdentifierPart::
+    UnicodeIDContinue
+    $
+    \ UnicodeEscapeSequence
+    <ZWNJ> <ZWJ>
+
+  UnicodeIDStart::
+    any Unicode code point with the Unicode property "ID_Start"
+
+  UnicodeIDContinue::
+    any Unicode code point with the Unicode property "ID_Continue"
+
+  NOTE 3
+  The sets of code points with Unicode properties "ID_Start" and
+  "ID_Continue" include, respectively, the code points with Unicode
+  properties "Other_ID_Start" and "Other_ID_Continue".
+
+template: productions
+features: [class-methods-private]
+---*/
+
+//- fields
+get #$() {
+  return this.#$_;
+}
+get #_() {
+  return this.#__;
+}
+get #\u{6F}() {
+  return this.#\u{6F}_;
+}
+get #℘() {
+  return this.#℘_;
+}
+get #ZW_‌_NJ() {
+  return this.#ZW_‌_NJ_;
+}
+get #ZW_‍_J() {
+  return this.#ZW_‍_J_;
+}
+
+//- privateinspectionfunctions
+$(value) {
+  this.#$_ = value;
```

Comment:
No such private field exists. You'll need to this: 

```
#$_; #__; #\u{6F}_; #℘_; #ZW_‌_NJ_; #ZW_‍_J_;
```
...under the `//- fields` label.

Or you can just `return 1;` from all of the getters and remove these assignments. 

Note that this applies to all of the case files in this changeset. I recommend running `./make.py build` to verify the resulting test material is valid.